client/src/components/podcasts/PodcastShowPage.js

```diff
@@ -16,15 +27,58 @@ const PodcastShowPage = (props) => {
       console.error(error.message)
     }
   }
-  
+
+  const postPodcastReview = async (reviewPayload) => {
+    try {
+      const response = await fetch(`/api/v1/podcasts/${podcastId}/reviews`, {
+        method: "POST",
+        headers: new Headers({
+          "Content-Type": "application/json"
+        }),
+        body: JSON.stringify(reviewPayload)
+      })
+      if (!response.ok) {
+        if (response.status === 422) {
+          const body = await response.json()
+          const newErrors = translateServerErrors(body.errors)
+          return setErrors(newErrors)
+        } else {
+          const errorMessage = `${response.status} (${response.statusText})`
+          const error = new Error(errorMessage)
+          throw (error)
+        }
+      } else {
+        const body = await response.json()
+        const updatedReviews = podcast.reviews.concat(body.review)
+        setPodcast({...podcast, reviews: updatedReviews })
+      }
+    } catch (error) {
+      console.error(`Error in fetch: ${error.message}`)
+    }
+  }
+
   useEffect(() => {
     fetchPodcast()
   }, [])
 
+  const reviews = podcast.reviews.map(review => {
+    return (
+      <ReviewTile 
+      key={review.id}
+      review={review}
+      />
+    )
+  })
+
   return (
     <div>
       <h1>{podcast.name}</h1>
       <p>{podcast.description}</p>
+      <ul>
+        {reviews}
+      </ul>
+      <ErrorList errors={errors} />
+      <PodcastReview postReview={postPodcastReview}/>
```

Comment:
Might consider moving your form for adding a review before you list the reviews - if we had 50 reviews, a user wouldn't even know there was a form buried at the bottom